src/EpicWithSpecialHeader/index.stories.tsx

```diff
@@ -8,20 +8,58 @@ import {
     buildEpicParagraphDocs,
 } from '../storybookCommon/argTypes';
 import { BrazeMessageProps } from '../Epic';
+import { grid, withGrid } from '../../.storybook/grid/withGrid';
 
 const NUMBER_OF_PARAGRAPHS = 9;
 const paragraphDocs = buildEpicParagraphDocs(NUMBER_OF_PARAGRAPHS);
 
 export default {
     component: 'EpicWithSpecialHeader',
     title: 'EndOfArticle/EpicWithSpecialHeader',
+    decorators: [withGrid],
```

Comment:
The grid image picker is a custom storybook plugin we wrote to help marketing interact with the Grid and build a signed `i.guim.co.uk` image URL. I think there are possibly elements of it which are hardcoded around an image size suitable for a banner which may not be appropriate here (i.e. generating images which are too large for the small header image size used by this component).

Reply:
I'm not too concerned about image size as it seems there's functionality in the grid plugin to create a new crop of the image if size is an issue. I think this can be solved by making sure people using the tool know about the option to create new crops

![Screenshot 2023-04-17 at 12 23 23](https://user-images.githubusercontent.com/5357530/232470918-5aad6d06-eec9-4025-808c-708fe9fd71e9.png)

The tool also gives a warning about over-small images, which can lead to poor quality image rendering in the browser/app

![Screenshot 2023-04-17 at 12 24 19](https://user-images.githubusercontent.com/5357530/232470959-166fdf9b-a967-4fdc-8913-a4cec37bab5e.png)

I did see an issue with using an image of inappropriate (non-square) images - they can appear behind the close button. Adding an `autofocus` attribute to the close button could help to make it more prominent?

![Screenshot 2023-04-17 at 12 17 39](https://user-images.githubusercontent.com/5357530/232471628-b6c41518-805c-4ca4-8b8c-4d71daf412f9.png)

![Screenshot 2023-04-17 at 12 18 18](https://user-images.githubusercontent.com/5357530/232471681-f958895f-1d03-478f-a5ca-f79188f94c2f.png)

At the end of the day, the question is: are we willing to let Marketing do these checks for themselves when skinning the component with data in Storybook? Or do we want to limit their choices in various ways by hardcoding them into the component's `canRender` checks?